gc.c

```diff
@@ -48,7 +48,9 @@
 #  define malloc_usable_size(a) malloc_size(a)
 # endif
 #else
```

Comment:
I think we should check for `HAVE_MALLOC_H` rather than `HAVE_MALLOC_TRIM`. We only check for `HAVE_MALLOC_TRIM` in `default.c` because we don't need to include `malloc.h` if we don't have `malloc_trim`.

```suggestion
#elif defined(HAVE_MALLOC_H)
# include <malloc.h>
#endif
```